app/controllers/assignments_controller.rb

```diff
@@ -108,6 +109,12 @@ def destroy
     end
   end
 
+  def allow_iframe_lti
+    return unless session[:isLTI]
+
+    response.headers.except! "X-Frame-Options"
```

Comment:
Can we use     `response.headers["X-FRAME-OPTIONS"] = "ALLOW-FROM http://example.com"` to white list just the moodle url if we can get moodle url in the session ?